src/HashBus.Twitter.Monitor.CatchUp/SearchTweetService.cs

```diff
@@ -3,26 +3,26 @@
     using System;
     using System.Collections.Generic;
     using System.Linq;
-    using HashBus.Twitter.Events;
+    using HashBus.Application.Commands;
     using Tweetinvi;
     using Tweetinvi.Core.Credentials;
     using Tweetinvi.Core.Parameters;
 
-    class TweetReceivedService : ITweetReceivedService
+    class SearchTweetService : ISearchTweetService

```

Comment:
I'm not really convinced by this name. I think the problem stems from the fact that we (I) gave this class an inappropriate responsibility, which is to map the tweets to commands. I suggest we move the call to the mapper to map tweets to commands out from this class and into the consumers of the `Get()` method. This would change the return type of `Get()` to `Tweetinvi.Core.Interfaces.ITweet` and would naturally lead to this class becoming `TweetService : ITweetService`.